### PR TITLE
eth: improve log message

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
also fixes a typo

### CODE REVIEW

**Review by DGKSK8LIFE — APPROVED** [review]
LGTM. Make sure that `ethPeersWithoutTransacion` isn't an actual method though.

File: eth/handler.go [hunk 1]
```diff
@@ -437,14 +437,14 @@ func (h *handler) BroadcastTransactions(txs types.Transactions, propagate bool)
...
-			peers := h.peers.ethPeersWithoutTransacion(tx.Hash())
+			peers := h.peers.ethPeersWithoutTransaction(tx.Hash())
 
 			// Send the block to a subset of our peers
 			transfer := peers[:int(math.Sqrt(float64(len(peers))))]
 			for _, peer := range transfer {
 				txset[peer] = append(txset[peer], tx.Hash())
 			}
-			log.Trace("Broadcast transaction", "hash", tx.Hash(), "recipients", len(peers))
+			log.Trace("Broadcast transaction", "hash", tx.Hash(), "recipients", len(txset))
```
Comment (holiman) [inline eth/handler.go:447]: That's not a typo fix -- pretty certain that changes the semantics of the message

Comment (MariusVanDerWijden) [reply eth/handler.go:447]: Yes, it changes the message from logging all peers to only logging the peers that we actually send the transaction to

Comment (holiman) [reply eth/handler.go:447]: Not quite, does it? The `len(txset)` will grow for each transaction.. ? 
So if `txs` are two transactoins, and tx1 is sent to `peerA`, then txset will have one key, `peerA`. 
If `tx2` is sent only to `peerB`, then the key `peerB` will be added to `txset`, and `len(txset)` will be `2`, not `1`. 
I think what you want is `len(transfer)`

Comment (MariusVanDerWijden) [reply eth/handler.go:447]: Ahh I meant to put `len(transfer)`, sorry

**Review by holiman — APPROVED** [review]

**Pull request merged**